examples/angular/src/BUILD.bazel

```diff
@@ -161,28 +163,33 @@ terser_minified(
     src = ":bundle-es5",
 )
 
-web_package(
+inject_html(
+    name = "inject_scripts_for_prod",
+    src = "//src:example/index.prod.html",
+    # we can't produce "example/index.html" since that collides with the devmode file
+    # pkg_web rule will re-root this path because it starts with _prodapp
```

Comment:
This is sneaky and I think needs a more comprehensive explanation for the purposes of an example. Maybe
```
# we can't produce "example/index.html" since that collides with the devmode file
# so we output "_prodapp/src/example/index.html" since pkg_web rule will re-root paths
# that start with _{name} by default which will put this file under `src/example/index.html`
# in the web package.
```
?

Reply:
sounds good